docs/framework/guides/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md

```diff
@@ -46,15 +46,18 @@ import { icons } from '@ckeditor/ckeditor5-core';
 
 export default class FormView extends View {
 	constructor( locale ) {
+		// Set the template.
```

Comment:
Actually, in the final code (https://github.com/ckeditor/ckeditor5-tutorials-examples/blob/main/abbreviation-plugin/part-3/abbreviation/abbreviationview.js) above the `this.focusTracker = new FocusTracker();` is only `super( locale );`, so I would change this to "View class constructor invoke" or something like that.